core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/PhysicalLinksCreator.java

```diff
@@ -1,44 +1,43 @@
-/*******************************************************************************
+/*******************************************************************************
  * Copyright (c) 2006, 2020 THALES GLOBAL SERVICES.
- * 
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License 2.0 which is available at
- * http://www.eclipse.org/legal/epl-2.0
- * 
- * SPDX-License-Identifier: EPL-2.0
- * 
- * Contributors:
- *    Thales - initial API and implementation
- *******************************************************************************/
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * 
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
 package org.polarsys.capella.core.projection.exchanges;
 
-import java.util.Collection;
```

Comment:
keep the original indentation here, too many diff lines.

Reply:
no change has been made here.. i don't know why it appears like that